filters/exclusions.txt

```diff
@@ -57,6 +57,7 @@ $popup,~third-party
 /^##\.adsbox$/
 /adframe.
 ! Excluding Google Contributor anti-adblock baits
+/^##\[class\^="div-gpt-ad"\]$/
```

Comment:
I guess it might also help here https://github.com/AdguardTeam/AdguardFilters/issues/78397 without `gigazine.net##body > div[class][style*="bottom: 0px; left: 0px;"][style*="z-index:"][style*="background-color:"][style*="display: flex; justify-content: center;"]`. What do you think?

Reply:
Yes, it should also fix these issues - https://github.com/AdguardTeam/AdguardFilters/issues/78400, https://github.com/AdguardTeam/AdguardFilters/issues/78390, but I think that leaving this rule is fine.
I'm also thinking about adding `||fundingchoicesmessages.google.com^$third-party` to AdGuard Annoyances filter, it should fixes a lot of cookie notice and anti-adblock issues.